sonar-plugin-api/src/main/java/org/sonar/api/component/mock/MockSourceFile.java

```diff
@@ -21,6 +21,10 @@
 
 import org.sonar.api.component.SourceFile;
 
+/**
+ * @deprecated since 5.6
+ */
+@Deprecated
 public class MockSourceFile implements SourceFile {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)